views/default/simplesaml/settings/service_provider_advanced.php

```diff
@@ -50,6 +50,19 @@
 	$body .= elgg_format_element('div', [], $external_id);
 }
 
+$force_authentication = $plugin->getSetting('force_authentication');
+if ($force_authentication == $source_id) {
+	// Only show when the current sourec has the force authentication enabled
+
+	$force_authentication_cidrs = elgg_echo('simplesaml:settings:sources:configuration:force_authentication_cidrs');
+	$force_authentication_cidrs .= elgg_view('input/text', [
+		'name' => "params[force_authentication_cidrs]",
+		'value' => $plugin->getSetting("force_authentication_cidrs"),
```

Comment:
hmm... you can configure cidrs not per service_provider... as you store it in a general setting... Shouldn't it be stored as part of the Service Provider (source_id)? This could cause issues switching to another provider as the cidr settings will still apply. This needs to be change before we can accept this